Tasks/Maven/maventask.ts

```diff
@@ -161,6 +171,20 @@ mvnGetVersion.exec()
         console.error("SonarQube analysis failed");
         sonarQubeRunFailed = true;
     })
+    .then(function (code) { // Pick up files from the Java code analysis tools
+        // The files won't be created if the build failed, and the user should probably fix their build first
+        if (userRunFailed) {
+            console.error('Could not retrieve code analysis results - Maven run failed.');
+            return;
+        }
+
+        codeAnalysis.uploadCodeAnalysisResults();
+    })
+    .fail(function (err) {
```

Comment:
Yep. The comment from above was accepted and resolved.

Resolved, no action required.